release/long_running_tests/run.sh

```diff
@@ -62,5 +62,6 @@ pip install --upgrade pip
 pip install -U tensorflow==1.14
 pip install -q -U "$wheel" Click
 pip install -q "ray[all]" "gym[atari]"
-python "workloads/$workload.py"
+cd ..
```

Comment:
which directory are we supposed to be in? why the cd ..?

Reply:
There is a line that calls `cd wrk`, so this fails to find the workloads/*.py file.